client/components/external-link/index.jsx

```diff
@@ -41,6 +42,10 @@ export default React.createClass( {
 			rel: 'external'
 		} );
 
+		if ( props.target ) {
```

Comment:
Do we need the `rel` if `target` is anything other than `_blank` ?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target